students/colephalen/lesson02/assignment/src/charges_calc.py

```diff
@@ -18,30 +23,106 @@ def load_rentals_file(filename):
     with open(filename) as file:
         try:
             data = json.load(file)
-        except:
+        except Exception as e:
```

Comment:
Try to avoid catching general Exceptions. You should look to find out what the exception thrown is and either try to handle it or report it to the user (like the print below).